tests/ssl-opt.sh

```diff
@@ -63,6 +63,7 @@ guess_config_name() {
 
 O_SRV="$OPENSSL_CMD s_server -www -cert data_files/server5.crt -key data_files/server5.key"
 O_CLI="echo 'GET / HTTP/1.0' | $OPENSSL_CMD s_client"
+G_PRI="--priority=NORMAL"
```

Comment:
Ok, this is used in two places, but I'm not sure it warrants a variable. If it does, it should be in a more local scope and with a more explicit name. It's hard to understand what this is for here, whereas it's easy to understand what `--priority=NORMAL` does in context.